proto/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -175,14 +176,16 @@ message ABCIQueryResponse {
   uint32 code = 1;
   // Deprecated: use "value" instead
   reserved 2;
-  string   log       = 3; // nondeterministic
-  string   info      = 4; // nondeterministic
-  int64    index     = 5;
-  bytes    key       = 6;
-  bytes    value     = 7;
-  ProofOps proof_ops = 8;
-  int64    height    = 9;
-  string   codespace = 10;
+  string log   = 3; // nondeterministic
+  string info  = 4; // nondeterministic
+  int64  index = 5;
+  bytes  key   = 6;
+  bytes  value = 7;
+  // depreacted in favor of comet proof type
+  reserved 8;
+  int64                       height    = 9;
+  string                      codespace = 10;
+  .tendermint.crypto.ProofOps proof_ops = 11;
```

Comment:
I thought I had dirt on my screen but it is actually a dot.
Is it expected?

Reply:
some places we do it and others we dont, im not sure which way to go. I also find it odd

Reply:
tested without the dot and it complains with linting